MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsRequestsManager.kt

```diff
@@ -99,13 +98,8 @@ internal class SensorsRequestsManager(application: Application) : SensorEventLis
 
                     // Hack for core implementation, expecting iOS-like rotation values,
                     // which remaps and reverses axises depending on orientation.
-                    // Since in Android sensor values does not depend on screen orientation we need to 'emulate' that.
                     orientations[0] = -orientations[0]
-                    if (screenOrientation == Surface.ROTATION_90) {
-                        orientations[2] = -orientations[2]
-                    } else if (screenOrientation == Surface.ROTATION_270) {
-                        orientations[1] = -orientations[1]
-                    }
+                    orientations[2] = -orientations[2]
```

Comment:
It wasn't actually needed, for both landscape and landscapeReverse we need to invert yaw and roll.